Sources/PackagePlugin/PluginMessages.swift

```diff
@@ -263,17 +265,18 @@ enum PluginToHostMessage: Codable {
         }
     
     /// The plugin defines a build command.
-    case defineBuildCommand(configuration: CommandConfiguration, inputFiles: [String], outputFiles: [String])
-    
+    case defineBuildCommand(configuration: CommandConfiguration, inputFiles: [URL], outputFiles: [URL])
+
     /// The plugin defines a prebuild command.
-    case definePrebuildCommand(configuration: CommandConfiguration, outputFilesDirectory: String)
+    case definePrebuildCommand(configuration: CommandConfiguration, outputFilesDirectory: URL)
     
         struct CommandConfiguration: Codable {
+            var version = 2
```

Comment:
Used this as a simple way to "version" the library. This allows us to detect cases where an old plugin library is being used with a newer SwiftPM which would otherwise silently exhibit broken behavior.